voting/src/general_meeting.rs

```diff
@@ -415,7 +415,7 @@ impl AsRef<StorageKeyRef> for GeneralMeetingId {
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(transparent)]
 pub struct TimeStamp {
-    time: u64,
```

Comment:
`Some part of implentation needs to be changed in order to implement tests.` does not give any information.
Let's remove it.
Also there is a typo. https://github.com/CodeChain-io/foundry/pull/478#discussion_r473687117